R/cleanup.R

```diff
@@ -342,7 +342,7 @@ remove_tails <- function(prot, by_column = "DomArch",
         # !! Insert line to read domains_keep
 
         # Contains all domains separated by "|"
-        domains_for_grep <- paste(domains_keep$domains, collapse = "|")
+        domains_for_grep <- paste(.data$domains_keep$domains, collapse = "|")
```

Comment:
It doesn't look like NSE is used here, so the .data prefix is not needed. However, we do need to figure out how `domains_keep` is defined. It may have originally been sourced from the Global Environment